### ISSUE TITLE
pdf插件无效 已经解决。相对路径是从source开始的。

### ISSUE THREAD

**Issue by ccvvv** [theme-next/hexo-theme-next#845]
### I agree and want to create new issue 

- [X ] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [ X] Yes, I was on [NexT Documentation Site](http://theme-next.org/docs/).
- [ X] And yes, I already searched for current [issues](https://github.com/theme-next/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 
添加pdf插件应该显示出pdf内容

### Actual behavior 
没有生效

### Steps to reproduce the behavior 
1. 添加插件 git clone https://github.com/theme-next/theme-next-pdf source/lib/pdf
2. 设置主题配置   
# PDF tag, requires two plugins: pdfObject and pdf.js
# pdfObject will try to load pdf files natively, if failed, pdf.js will be used.
# The following `cdn` setting is only for pdfObject, because cdn for pdf.js might be blocked by CORS policy.
# So, YOU MUST install the dependency of pdf.js if you want to use pdf tag and make it work on all browsers.
# See: https://github.com/theme-next/theme-next-pdf
pdf:
  enable: true
  # Default height
  height: 500px
  pdfobject:
    # Use 2.1.1 as default, jsdelivr as default CDN, works everywhere even in China
    cdn: //cdn.jsdelivr.net/npm/pdfobject@2.1.1/pdfobject.min.js
    # CDNJS, provided by cloudflare, maybe the best CDN, but not works in China
    #cdn: //cdnjs.cloudflare.com/ajax/libs/pdfobject/2.1.1/
3，配置pdf文件。
我将文件放在 /source/c.pdf
在hello-word.md文件中插入以下代码
{% pdf ./c.pdf %}
仓库地址：https://github.com/ccvvv/ccvvv.github.io/tree/bf/themes/next/source
 Link to demo site with this issue: N/A
* Link(s) to source code or any usefull link(s): N/A

### Node.js and NPM Information

```

```

### Package dependencies Information

```

```

#### Hexo version

```

```

#### Hexo Configuration

```yml

```

### NexT Information

**NexT Version:**

- [ ] Latest Master branch
- [ ] Latest Release version
- [ ] Old version

**NexT Scheme:**

- [ ] All schemes
- [ ] Muse
- [ ] Mist
- [ ] Pisces
- [ ] Gemini

#### NexT Configuration:

```yml

```

**Comment by stevenjoezhang** [comment]
{% pdf /c.pdf %}

**Comment by ccvvv** [comment]
> {% pdf /c.pdf %}

谢谢！试过了成功显示出一片空白，可能是因为手机不支持显示内嵌pdf吧。

**Comment by stevenjoezhang** [comment]
理论上大部分手机是支持的，如果有其它问题欢迎提出

**Comment by physchtest** [comment]
我的安卓、iPad等移动端设备都无法显示我博客上的pdf

**Comment by physchtest** [comment]
电脑端倒是可以正常显示。请问有解决方法吗？
